# Actual Reference frame (based on request and sonar availability)
uint8 actual_ref_frame
# Target Reference frame (only based on the request)
uint8 target_ref_frame